.github/workflows/tests.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         with:
           node-version: 16.x
           cache: 'yarn'
+      - run: curl https://raw.githubusercontent.com/Tenderly/tenderly-cli/master/scripts/install-linux.sh | sudo sh
```

Comment:
Why do we need this on every task? I believe tests are still running on the Alchemy fork right?

Reply:
it was a quick fix to silence some very loud tenderly logs that show up if the plugin is included in the hardhat config, but the user is not logged in via the cli. ill see if there is a better way to silence them

Reply:
rolled back these additions and now the SILLY SERVICE logs seem to have disappeared...